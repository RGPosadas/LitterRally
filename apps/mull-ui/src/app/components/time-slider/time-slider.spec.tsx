import React from 'react';
import { render } from '@testing-library/react';

import TimeSlider from './time-slider';

describe('TimeSlider', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<TimeSlider />);
    expect(baseElement).toBeTruthy();
  });
});
