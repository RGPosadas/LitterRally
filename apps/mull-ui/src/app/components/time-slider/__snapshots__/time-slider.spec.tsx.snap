// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTimePicker should match snapshot 1`] = `
<div
  className="custom-slider-container"
>
  <div
    className="custom-text-input-label"
  >
    Text
     
    12:00 pm
  </div>
  <div
    className="rc-slider custom-slider-style"
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="rc-slider-rail"
      style={
        Object {
          "backgroundColor": "#27b09a",
          "height": 10,
        }
      }
    />
    <div
      className="rc-slider-track"
      style={
        Object {
          "backgroundColor": "silver",
          "height": 10,
          "left": "0%",
          "right": "auto",
          "width": "50.526315789473685%",
        }
      }
    />
    <div
      className="rc-slider-step"
    />
    <div
      aria-disabled={false}
      aria-valuemax={23.75}
      aria-valuemin={0}
      aria-valuenow={12}
      className="rc-slider-handle"
      onBlur={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      role="slider"
      style={
        Object {
          "backgroundColor": "white",
          "borderColor": "green",
          "height": 28,
          "left": "50.526315789473685%",
          "marginTop": -9,
          "right": "auto",
          "transform": "translateX(-50%)",
          "width": 28,
        }
      }
      tabIndex={0}
    />
    <div
      className="rc-slider-mark"
    />
  </div>
  <span
    className="error-message"
  >
    Time is not valid.
  </span>
</div>
`;
